apps/src/dance/ai/cdoFieldDanceAi.js

```diff
@@ -123,35 +96,41 @@ class CdoFieldDanceAi extends GoogleBlockly.Field {
         fill: color.neutral_dark90,
         x: 1,
         y: 1,
-        width: textWidth,
-        height: FIELD_HEIGHT,
+        width: 3 * ITEM_WIDTH + 2 * FIELD_PADDING,
+        height: FIELD_HEIGHT + 2,
         rx: 3,
       },
       this.backgroundElement
     );
 
+    inputs?.forEach((input, index) => {
+      GoogleBlockly.utils.dom.createSvgElement(
```

Comment:
This is old, but we can't just use the global `Blockly` import here?

Reply:
Good question.  We can, and that will use the `BlocklyWrapper`.  We also refer to `GoogleBlockly.Field` in this file, so I'm curious if it's appropriate to reference both `Blockly` and `GoogleBlockly` in the same file, for different purposes.  (Maybe @code-tools have thoughts too.)

Reply:
As a general rule I think we should use the wrapper. The reason I can think of to not use it is if we want to use the google blockly version of a parameter/function/etc. and we have our own custom version. But I think that should rarely happen outside of the wrapper class itself.